source/bao_hyp/config.rst

```diff
@@ -65,41 +65,512 @@ describe the image running on the VM (see `Guest Image`_);
+            } iommu;
+
+            struct {
+                paddr_t base;
+            } aclint_sswi;
+        };
+
+    In case the available interrupt controller is the legacy PLIC:
+
+    - ``irqc.plic.base`` [mandatory if PLIC is available] - is the base address for the PLIC;
+
+    In case the available interrupt controller is an AIA containing an APLIC:
+
+    - ``irqc.aia.aplic.base`` [mandatory if APLIC is available] - is the base address for the APLIC;
+
+    When an IOMMU is available:
+
+    - ``iommu.base`` [mandatory if IOMMU is available] - is the base address for the IOMMU;
+    - ``iommu.fq_irq_id`` [mandatory if IOMMU is available] - the Fault Queue interrupt ID (the
+      current implementation assumes this is a wired interrupt);
 
 CPU Affinity
 ************
 
+The configuration file of the Bao hypervisor also enables the definition of core affinity, which
+involves selecting the physical core where the guest should run.
+
+.. figure:: img/cpu-affinity.svg
+    :align: center
+    :width: 100%
+    :name: cpu-affinity-fig
+
+|
+
+This functionality is achieved through the following configuration parameter:
+
+- ``cpu_affinity`` [optional] - corresponds to a bitmap signaling the preferred physical CPUs
+  assigned to the VM. If this value is mutually exclusive for all the VMs, the physical CPUs
+  assigned to each VM follow the bitmap. Otherwise (in case of bit overlap or lack of affinity
+  definition), the CPU assignment is defined by the hypervisor;
+
 Coloring
 ********
 
+Cache coloring is a technique used to partition shared Last Level Cache (LLC) sets among different
+guests (i.e., VMs). The main goal is to minimize cache conflicts and enhance overall system
+performance by carefully assigning specific colors, representing cache sets, to different entities.
+Bao supports cache coloring which can be configured using the ``colors`` field within the
+``vm_config`` struct.
+
+- ``colors`` [optional] - corresponds to a bitmap for the assigned cache colors of the VM. This
+  value is truncated depending on the number of available colors calculated at run-time, i.e., its
+  platform-dependent. By default, the coloring mechanism is not active. For instance, the following
+  picture depicts a hypothetical setup with a 50/50 coloring scheme;
+
+.. figure:: img/llc-colors.svg
+    :align: center
+    :width: 100%
+    :name: llc-colors-fig
+
+|
+
+It's important to note that cache coloring relies on the careful assignment of colors to each VM.
+However, this mechanism may not take effect if the physical mapping feature is enabled for a
+specific memory region. Cache coloring exclusively operates in virtual memory systems, i.e.,
+systems featuring Memory Management Units (MMUs) for address translation.
+
 Shared Memory Configuration
 ---------------------------
+.. _Shared Memory Configuration:
+
+In Bao's configuration multiple (``shmem0``, ``shmem1``, ...) or no shared memory can be set. An ID
```

Comment:
```suggestion
In Bao's configuration, you can set multiple shared memory regions (e.g., shmem0, shmem1) or none at all.
```

Reply:
Thanks, fixed in commit https://github.com/bao-project/bao-docs/pull/78/commits/3f2a6f174933062acb335c27d1a70adca084d5bc!